content/docs/command-reference/exp/run.md

```diff
@@ -84,13 +84,22 @@ steps as `make_checkpoint()` — please refer to its reference for details.
 You can now use `dvc exp run` to begin the experiment. If the process gets
 interrupted (e.g. with `[Ctrl] C` or by an error), all the checkpoints so far
 will be preserved. When a run finishes normally, a final checkpoint will be
-added (if needed) to wrap up the experiment.
+added (if needed) to wrap up the experiment. Subsequent uses of `dvc exp run`
+will resume from this point (using the latest cached versions of all outputs).
 
-Following uses of `dvc exp run` will continue from this point (using the latest
-cached versions of all outputs). You can add a `--rev` to continue from a
-previous checkpoint instead (list them with `dvc exp show`). Or use `--reset` to
-start over (discards previous checkpoints and deletes `checkpoint` outputs, like
-the first `dvc exp run`) — useful for re-training ML models, for example.
+List previous checkpoints with `dvc exp show`. To continue from a previous
+checkpoint, you must first `dvc exp apply` it before using `dvc exp run`. For
```

Comment:
The first sentence still feels a bit disconnected. How about something like
```suggestion
List previous checkpoints with `dvc exp show`. To continue from any of
them, you must first `dvc exp apply` it before using `dvc exp run`.
```

Reply:
"them -> it" sounds awkward - I'd prefer slight repetition instead.

Reply:
True 👍